database/migrations/2022_04_06_000000_create_blog_article_table.php

```diff
@@ -16,10 +16,11 @@ public function up()
             $table->string('slug');
             $table->string('layout')->nullable();
             $table->unsignedInteger('footer_id')->nullable();
-            $table->boolean('indexable');
+            $table->boolean('indexable')->nullable();
```

Comment:
I would argue that this needs to have a value, and you have a few options on how to set a default.

You can set a default here in the migration as we do for the Shopify products: `$table->boolean('indexable')->default(true);`

You can also do as we did with the pages, ensure a value is set in the [Form model](https://github.com/astrogoat/strata/blob/8fcab0f393047794cdd272ac594ff7c2d7281bb2/src/Http/Livewire/Models/Pages/Form.php#L41): `$this->page->indexable = true;`